x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/authz/RBACEngineTests.java

```diff
@@ -1527,14 +1526,14 @@ public void testExplicitMappingUpdatesAreNotGrantedWithIngestPrivileges() {
 
         PutMappingRequest request = new PutMappingRequest("*");
         request.source("{ \"properties\": { \"message\": { \"type\": \"text\" } } }", XContentType.JSON);
-        Set<String> authorizedIndices = RBACEngine.resolveAuthorizedIndicesFromRole(
+        AuthorizedIndices authorizedIndices = RBACEngine.resolveAuthorizedIndicesFromRole(
             role,
             getRequestInfo(request, PutMappingAction.NAME),
-            lookup
+            lookup,
+            () -> ignore -> {}
         );
-        // The authorized indices is the lazily loading set implementation
-        assertThat(authorizedIndices, instanceOf(RBACEngine.AuthorizedIndicesSet.class));
-        assertThat(authorizedIndices.isEmpty(), is(true));
+        assertThat(authorizedIndices, instanceOf(RBACEngine.AuthorizedIndices.class));
```

Comment:
Same here. The assertion is not needed anymore.

Reply:
Pushed [531245e](https://github.com/elastic/elasticsearch/pull/91618/commits/531245e18824215ec0f6f316a0bfc508af2bd895)